src/pages/user/viewer/positions.jsx

```diff
@@ -1,103 +1,133 @@
+			<TableCell flex="2"         mobileHide>{position.event_name??<b>Permanent stilling</b>}</TableCell>
+		</SelectableTableRow>
+	)
 }
 
 
-export const UserViewerPositions = ({ user, reload: reloadUser }) => {
-    const [ loading, setLoading ] = useState(false);
-
-    const [ currentEvent, setCurrentEvent ] = useState(null);
-
-    const [visibleUUIDPositions, setVisibleUUIDPositions] = useState(false);
-
-    const reload = async () => {
-        setLoading(true);
-        setCurrentEvent(await getCurrentEvent());
-        setLoading(false);
-    }
-
-    useEffect(() => {
-        reload();
-    }, []);
-
-    if(loading) {
-        return (<PageLoading />)
-    }
-    return (
-        <>
-            <InnerContainer mobileHide border extramargin>
-                <InputCheckbox label="Vis UUID" value={visibleUUIDPositions} onChange={() => setVisibleUUIDPositions(!visibleUUIDPositions)} />
-            </InnerContainer>
-            <InnerContainer extramargin>
-                <InnerContainerTitle>Nåværende Stillinger</InnerContainerTitle>
-                <Table>
-                    <TableHead border>
-                        <TableRow>
-                            <TableCell flex="1" visible={!visibleUUIDPositions}>UUID</TableCell>
-                            <TableCell flex="2">Navn</TableCell>
-                            <TableCell flex="0 24px" />
-                        </TableRow>
-                    </TableHead>
-                    <TableBody>
-                        <PositionList canRemove={true} reload={reloadUser} show_uuid={visibleUUIDPositions} position_mappings={user.position_mappings.filter(mapping => !mapping.event_uuid || mapping.event_uuid == currentEvent?.uuid )} />
-                    </TableBody>
-                </Table>
-            </InnerContainer>
-            <InnerContainer extramargin mobileHide />
-            <InnerContainer extramargin>
-                <InnerContainerTitle>Tidligere Stillinger</InnerContainerTitle>
-                <Table>
-                    <TableHead border>
-                        <TableRow>
-                            <TableCell flex="1" visible={!visibleUUIDPositions}>UUID</TableCell>
-                            <TableCell flex="2">Navn</TableCell>
-                            <TableCell flex="0 24px" />
-                        </TableRow>
-                    </TableHead>
-                    <TableBody>
-                        <PositionList canRemove={false} reload={reloadUser} show_uuid={visibleUUIDPositions} position_mappings={user.position_mappings.filter(mapping => mapping.event_uuid && mapping.event_uuid != currentEvent?.uuid )} />
-                    </TableBody>
-                </Table>
-            </InnerContainer>
-        </>
-    )
+
+export const UserPositions = () => {
+
+	// Import the following React contexts:
+	const windowManager = useContext(WindowManagerContext);
+
+	const { uuid } = useParams();
```

Comment:
By using `useParams` to get the user(and then re-fetching it) you are wasting more network resources, and you also made the component less re-usable. If `userPositions` is simply a component that shows the positions of an user, it is better that you can simply just pass that user in if it is only used in cases where you already have access to the user.